sql/plan/resolved_table.go

```diff
@@ -0,0 +1,106 @@
+package plan
+
+import (
+	"io"
+
+	"gopkg.in/src-d/go-mysql-server.v0/sql"
+)
+
+// ResolvedTable represents a resolved SQL Table.
+type ResolvedTable struct {
+	sql.Table
+	name string
+}
+
+var _ sql.Table = (*ResolvedTable)(nil)
+var _ sql.Node = (*ResolvedTable)(nil)
+var _ sql.Nameable = (*ResolvedTable)(nil)
+
+// NewResolvedTable creates a new instance of ResolvedTable.
+func NewResolvedTable(name string, table sql.Table) *ResolvedTable {
```

Comment:
If table has name, why do we need to set a name on resolvedTable too?

Reply:
`mem.Table` has name, the interface `sql.Table` hasn't it.

Reply:
Maybe `sql.Table` should have a name